homeassistant/components/evohome/__init__.py

```diff
@@ -654,6 +658,7 @@ def current_temperature(self) -> float | None:
         assert isinstance(self._evo_device, evo.HotWater | evo.Zone)  # mypy check
 
         if self._evo_broker.temps.get(self._evo_id) is not None:
```

Comment:
Side note: If we store the value in a walrus variable we don't need to look it up twice.